matcher/service/openreview_interface.py

```diff
@@ -130,12 +124,20 @@ def scores_by_type(self):
                 if 'translate_map' in score_spec
             }
 
+            score_defaults = {
+                inv_id: score_spec.get('default', 0) \
+                for inv_id, score_spec in scores_specification.items()
+            }
+
             self._scores_by_type = {
                 inv_id: [
                     (
                         edge['head'],
                         edge['tail'],
-                        self._edge_to_score(edge, translate_map=translate_maps.get(inv_id))
+                        self._edge_to_score(
+                            edge, 
+                            score_default=score_defaults.get(inv_id), 
```

Comment:
the default in the score spec is intended to be used when there is not edge between a specific pair reviewer-paper. 

The issue is in the Encode that used default = 0 when there is no score:

https://github.com/openreview/openreview-matcher/blob/master/matcher/encoder.py#L94

I think the function `_encode_scores` should put a default value when there is no edge between a reviewer-paper. 

Could you also add a test to check the default value of score spec is not being used and then implement the solution?

This is the best way to prove the issue has been fixed.

Reply:
Thanks for explaining this @melisabok . I have made the changes in a different PR at https://github.com/openreview/openreview-matcher/pull/181. Closing this PR.